### PR TITLE
Replaced Material UI components in UpdateFacilityMiddleware.ts

### LINKED ISSUE #4961: Replace Material UI components in `src/Components/Facility/UpdateFacilityMiddleware.tsx`:
```tsx
1: import { Card, CardContent } from "@material-ui/core";
```

### Tasks

Ensure all the below tasks are accomplished if applicable to the file being modified in the PR.

- [ ] Migrate all Material UI components to use new Care UI / custom components. (ref: [🛠️ Tailwinding Care](https://github.com/coronasafe/care_fe/issues/3742))
- [ ] Migrate all Material UI icons to use [🏥 **Care Icons**](https://github.com/coronasafe/care_fe/pull/4071).
- [ ] Migrate `PageTitle` to `Page` component if any. (See [example](https://github.com/coronasafe/care_fe/pull/4796/files#diff-8e39f154ccf13c9dedd0f968d21a0abbde450e88df9724a4e7d4264d487a7846))

For more information see the epic issue that tracks this issue.

### PR DESCRIPTION
**Pull request by GokulramGHV**
## Proposed Changes

- Fixes #4961
- Replaced Material UI components with custom components in `src/Components/Facility/UpdateFacilityMiddleware.ts`

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

### Screenshot
image

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

File: src/Components/Common/components/CardV2.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  className?: string;
+  raised?: "sm" | "md" | "lg" | "xl" | "2xl" | "inner";
+  border?: boolean;
+  rounded?: "sm" | "md" | "lg" | "xl" | "2xl" | "3xl";
+}) {
+  return (
+    <div
+      className={`${props.className}  bg-white p-5 ${
+        props.rounded ? `rounded-${props.rounded}` : "rounded-lg"
+      } ${props.raised ? `shadow-${props.raised}` : "shadow-md"} ${
```
Comment (rithviknishad) [inline src/Components/Common/components/CardV2.tsx]: @GokulramGHV string interpolating tailwind classes isn't nice. tailwind purge will not be able to detect the usages correctly. include full tailwind classnames.

**Comment by nihal467** [comment]
@GokulramGHV i can see a change requested by @rithviknishad , once the PR is final for testing, change the label to "Need testing"

**Review by rithviknishad — APPROVED** [review]
Looks way cleaner! 🚀

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**